# Generated by Django 4.1.2 on 2022-12-17 11:42

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
import utils.rename


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0175_auto_20221116_0142'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='avatar',
            field=models.ImageField(blank=True, null=True, upload_to=utils.rename.Rename.rename, verbose_name='Аватар'),
        ),
        migrations.AlterField(
            model_name='user',
            name='bg',
            field=models.ImageField(blank=True, null=True, upload_to=utils.rename.Rename.rename, verbose_name='Задний фон'),
        ),
        migrations.AlterField(
            model_name='user',
            name='password',
            field=models.CharField(default='pbkdf2_sha256$390000$T5iwjSsRJnOZQQh0PEZ7D8$3f2MZndJ4H05x3F84qgOHz1/OonixtdtSFBM5qZ4YPI=', max_length=128, verbose_name='Пароль'),
        ),
        migrations.AlterField(
            model_name='userimage',
            name='image',
            field=models.ImageField(blank=True, upload_to=utils.rename.Rename.rename, verbose_name='Изображение'),
        ),
        migrations.CreateModel(
            name='SaveContactCount',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('count', models.IntegerField(default=1)),
                ('created_at', models.DateTimeField(auto_now_add=True)),
                ('user', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='save_contact_user', to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
